File: Solution/main/java/com/example/programingtext/MainActivity.java
package com.example.programingtext;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TableLayout;
import android.widget.TableRow;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TableLayout t1=findViewById(R.id.table);
        TableRow row=new TableRow(this);
        TextView tv= new TextView(this);
        tv.setText("this is textview.");

        t1.addView(row);
        row.addView(tv);

    }
}

File: Module5/6-6-Service,Receiver,Admob/MainActivity.kt
package com.example.myapplication

import android.content.Intent
import android.content.IntentFilter
import android.media.MediaPlayer
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity()
{
    lateinit var btn1:Button
    lateinit var btn2:Button
    lateinit var mediaPlayer: MediaPlayer
    var url="https://vyasprakruti.000webhostapp.com/demo/iphone_13_original.mp3"
    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn1 = findViewById(R.id.btn1)
        btn2 = findViewById(R.id.btn2)
        //mediaPlayer = MediaPlayer.create(applicationContext,R.raw.iphone)

        //online


        var receiver = MyReceiver()
        var filter = IntentFilter()
        filter.addAction(Intent.ACTION_AIRPLANE_MODE_CHANGED)
        registerReceiver(receiver,filter)



        btn1.setOnClickListener {

           /* var i = Intent(applicationContext,MyService::class.java)
            startService(i)*/
            mediaPlayer.start()

            if(mediaPlayer==null)
            {
                mediaPlayer = MediaPlayer()
                mediaPlayer.setDataSource(url)
                mediaPlayer.start()
            }
        }
        btn2.setOnClickListener {
          /*  var i = Intent(applicationContext,MyService::class.java)
            stopService(i)*/

            mediaPlayer.stop()
        }

    }
}